pkg/common/morpc/codec.go

```diff
@@ -34,6 +34,10 @@ var (
 	defaultMaxMessageSize = 1024 * 1024 * 100
 )
 
+var (
+	DefaultMaxMessageSize = defaultMaxMessageSize
```

Comment:
Come on, make the original one capital.   What this kind of var buys you, other than more bugs?